app/browser/tabs.js

```diff
@@ -196,9 +196,10 @@ const updateAboutDetails = (tab, tabValue) => {
   } else if (location === 'about:history') {
     if (!history) {
       appActions.populateHistory()
+    } else {
+      tab.send(messages.HISTORY_UPDATED, history.toJS())
+      tab.send(messages.SETTINGS_UPDATED, appSettings.toJS())
```

Comment:
Might be safer to always send appSettings regardless of if history is falsy or not?